### ISSUE TITLE
OpenCascade headers

### ISSUE THREAD

**Issue by Saiel** [IfcOpenShell/IfcOpenShell#3164]
With default building and installation there are only IOS headers and libs in _installed directory, but also IOS uses OpecCascade headers (such as `gp_GTrsf.hxx`). As I understand, OpenCascade libs are linked statically already so I don't have to link them myself. But where I should take its headers? Can I just take `_deps-vs2022-x64-installed\opencascade-7.5.3\inc` directory? Or should I install OpenCascade 7.5.3 and use them from installation? Which way is supposed to be used?

Windows 10, vs22, x64

**Comment by aothms** [comment]
> Can I just take _deps-vs2022-x64-installed\opencascade-7.5.3\inc directory?

Yes, definitely. Same for the other libs in `_deps-vs2022-x64-installed`. Boost is an exception and in include: `_deps\boost_1_81_0` lib: `_deps\boost_1_81_0\stage\vs2017-x64\lib` on my system.

I'm quite proud of our build scripts 😎, but we have al long way to go in terms of setting up a good developer environment for implementers using ifopsh as a C++ dependency. Anything you want to share or contribute in this process is very welcome :)

**Comment by Saiel** [comment]
I think it's good to copy dependency headers to installation folder too. It'll make more clear what include directories should be added to a project, because I (at least) expect installation folder to be self-sufficient unless the opposite was explicitly stated.

Yeah, your build scripts are impressive :) I have some ideas for unify them under cmake interface, but first I want to test it myself.

**Comment by aothms** [comment]
I agree. It's also most similar to the way the nix build script is structured. I wouldn't want to disrupt people's build environments though, so we should change this is v0.8.0 onwards.

I would propose something like `./_build/win/vs2022-x64/build/occt-x.y.z/` and `./_build/win/vs2022-x64/install/occt-x.y.z/`. That's a bit like the current nix build-script output. Thoughts?

**Comment by Saiel** [comment]
> I would propose something like ./_build/win/vs2022-x64/build/occt-x.y.z/ and ./_build/win/vs2022-x64/install/occt-x.y.z/
No, I mean something like moving content of `./_deps-vs2022-x64-installed` to `./_installed-vs2022-x64/deps/`

However, I just now noticed that it is already called `*installed`, so it's my bad, I should have noticed this earlier and not ask about it. Though, it would be nice to have some tips in documentation on how to add IOS and its dependencies into user projects after building.
